.yardopts

```diff
@@ -0,0 +1 @@
+--markup markdown
```

Comment:
Where is this coming from?

Reply:
That was generated when I ran the convert

Reply:
Great, it actually comes from here: https://github.com/puppetlabs/pdk-templates/blob/master/config_defaults.yml#L57
Thanks to @HelenCampbell for pointing this out.

Reply:
this is for Puppet Strings/Yard in case anyone's curious. It tells Yard that the source documentation should be interpreted as markdown, not simply plain text.